README.md

```diff
@@ -198,12 +198,10 @@ Actions can be used for pre or post release parts.
 * `vcs-tag`: Tag the last commit
 * `vcs-publish`: Publish the changes (commits and tags)
 * `composer-update`: Update the version number in a composer file
-* `update-version-class`: Update the version constant in a class file.
-    * Option `class`: path to class to be updated, or fully qualified class name of the class containing the version constant
-    * Option `pattern`: optional, use to specify the string replacement pattern in your
-      version class. %version% will be replaced by the current / next version strings.
-      For example you could use `const VERSION = '%version%';`. If you do not specify
-      this option, every occurrence of the version string in the file will be replaced.
+* `files-update`: Update the version in one or multiple files. For each file to update, please provide an array with 
+    * Option `file`: path to the file to update
+    * Option `pattern`: optional, use to specify the string replacement pattern in your file. For example: 
+    `const VERSION = '%version%';`
```

Comment:
@Jonathanm10 This is not completely clear what the user have to configure. Maybe we should add an example into https://github.com/liip/RMT#configuration-examples WDYT?